collections/ansible_collections/purestorage/flashblade/docs/purefb_snmp_mgr.rst

```diff
@@ -0,0 +1,151 @@
+
+purefb_snmp_mgr -- Configure FlashBlade SNMP Managers
+=====================================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Synopsis
+--------
+
+Manage SNMP managers on a Pure Storage FlashBlade.
+
+This module is not idempotent and will always modify an existing SNMP manager due to hidden parameters that cannot be compared to the play parameters.
+
+
+
+Requirements
+------------
+The below requirements are needed on the host that executes this module.
+
+- python >= 2.7
+- purity_fb >= 1.1
+
+
+
+Parameters
+----------
+
+  fb_url (optional, str, None)
+    FlashBlade management IP address or Hostname.
+
+
+  name (True, str, None)
+    Name of SNMP Manager
+
+
+  notification (optional, str, trap)
+    Action to perform on event.
+
+
+  state (optional, str, present)
+    Create or delete SNMP manager
+
+
+  community (optional, str, None)
+    SNMP v2c only. Manager community ID. Between 1 and 32 characters long.
+
+
+  privacy_protocol (optional, str, None)
+    SNMP v3 only. Encryption protocol to use
+
+
+  auth_protocol (optional, str, None)
+    SNMP v3 only. Hash algorithm to use
+
+
+  host (optional, str, None)
+    IPv4 or IPv6 address or FQDN to send trap messages to.
+
+
+  version (optional, str, None)
+    Version of SNMP protocol to use for the manager.
```

Comment:
I believe this should be `version (optional, str, v3)`, since if no version is specified, v3 will be assumed

Reply:
Actually this comment made me realize I had left a logic hole. Filled in the latest push